.github/workflows/test.yml

```diff
@@ -9,6 +9,21 @@ on:
       - '.gitignore'
 
 jobs:
+  tests:
```

Comment:
📝 Consider adding a description for the 'tests' job to clarify its purpose in the workflow.